tests/testthat/test-convert_mae_se_assay_to_dt.R

```diff
@@ -101,6 +101,7 @@ test_that("convert_mae_assay_to_dt works as expected", {
   mae <- MultiAssayExperiment::MultiAssayExperiment(experiments = list("single-agent" = se))
   
   dt <- convert_mae_assay_to_dt(mae = mae, assay_name = "RefGRvalue", include_metadata = FALSE)
+  checkmate::expect_class(dt, c("data.frame", "data.table"))
```

Comment:
Better: `checkmate::expect_data_frame(dt)`

Reply:
No, it will not work correctly as we want to throw an error for data.frame which will not happen with `checkmate::expect_data_frame(dt)`. 
One option might be to change to `checkmate::expect_data_table(dt)`. See https://jira.gene.com/jira/browse/GDR-1961 for more details.

Reply:
Ok, thanks for the explanation! So maybe better to use `checkmate::expect_data_table` ?

Reply:
great idea. I'll do it that way.

Reply:
done